docker-compose/run_selenium_tests_local_keybase.sh

```diff
@@ -0,0 +1,213 @@
+keybase_env="${keybase_env_path}/${KEYBASE_ENV_FILE}"
+
+echo_msg " - Sourcing ENV secrets from: ${keybase_env}"
+echo_msg
+
+# Check that ENV file exists in correct location
+if [ ! -f "${keybase_env}" ]
+then
+echo_msg
+echo_msg "ERROR: The ENV secrets could NOT be found at: ${keybase_env}"
+echo_msg
+exit 1
+fi
+
+# Source ENVs
+source "${keybase_env}"
+
+# Check ENV vars have been sourced
+if [ -z "${DJANGO_USER_ID_SALT}" ]
+then
+echo_msg "ERROR: The DJANGO_USER_ID_SALT variable was not sourced!"
+exit 1
+fi
+if [ -z "${DJANGO_USER_ID_ITERATIONS}" ]
+then
+echo_msg "ERROR: The DJANGO_USER_ID_ITERATIONS variable was not sourced!"
+exit 1
+fi
+
+# Check temp certstore dir and create if not existing
+if [ -d "${CERTSTORE_TEMPORARY_MOUNT_PATH}" ]
+then
+echo_msg
+echo_msg "  - OK: The temporary certstore mount path is found at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+else
+mkdir ${CERTSTORE_TEMPORARY_MOUNT_PATH}
+echo_msg
+echo_msg "  - OK: Created the temporary certstore mount path at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+fi
+
+# Keybase cert files
+keybase_certfiles="${keybase_env_path}/${KEYBASE_CERTFILES_SUBPATH}"
+keybase_cert_file="${keybase_certfiles}/${CERT_FILENAME}"
+keybase_key_file="${keybase_certfiles}/${KEY_FILENAME}"
+
+# Check that certfiles in keybase dir exist
+if [ -f "${keybase_cert_file}" ]
+then
+    echo_msg
+    echo_msg "  - OK: The cert file was found at: ${keybase_cert_file}"
+else
+    echo_msg
+    echo_msg "ERROR: The cert file could NOT be found at: ${keybase_cert_file}"
+    exit 1
+fi
+
+if [ -f ${keybase_key_file} ]
+then
+    echo_msg
+    echo_msg "  - OK: The key file was found at: ${keybase_key_file}"
+else
+    echo_msg
+    echo_msg "ERROR: The key file could NOT be found at: ${keybase_key_file}"
+    exit 1
+fi
+
+# Copy certfiles from KeyBase to local for container mount
+echo_msg "  - COPY certfiles from KeyBase to local temp for container mount..."
+echo_msg
+cp ${keybase_cert_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+cp ${keybase_key_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
+
+# stop all before run selenium tests
+docker-compose -f docker-compose.selenium.yml down
+
+export DJANGO_USER_ID_SALT=${DJANGO_USER_ID_SALT}
+export DJANGO_USER_ID_ITERATIONS=${DJANGO_USER_ID_ITERATIONS}
+export DJANGO_PASSWORD_HASH_ITERATIONS=${DJANGO_PASSWORD_HASH_ITERATIONS}
+export DJANGO_SLSX_CLIENT_ID=${DJANGO_SLSX_CLIENT_ID}
+export DJANGO_SLSX_CLIENT_SECRET=${DJANGO_SLSX_CLIENT_SECRET}
+export HOSTNAME_URL=${HOSTNAME_URL}
+export USE_MSLSX=${USE_MSLSX}
+export DJANGO_MEDICARE_SLSX_REDIRECT_URI=${DJANGO_MEDICARE_SLSX_REDIRECT_URI}
+export DJANGO_MEDICARE_SLSX_LOGIN_URI=${DJANGO_MEDICARE_SLSX_LOGIN_URI}
+export DJANGO_SLSX_HEALTH_CHECK_ENDPOINT=${DJANGO_SLSX_HEALTH_CHECK_ENDPOINT}
+export DJANGO_SLSX_TOKEN_ENDPOINT=${DJANGO_SLSX_TOKEN_ENDPOINT}
+export DJANGO_SLSX_SIGNOUT_ENDPOINT=${DJANGO_SLSX_SIGNOUT_ENDPOINT}
+export DJANGO_SLSX_USERINFO_ENDPOINT=${DJANGO_SLSX_USERINFO_ENDPOINT}
```

Comment:
I notice this is just export definitions on variables that already exist.  Can you just export at definition time or is there some specific reason why it works this way?